spec/model_spec.cr

```diff
@@ -84,4 +84,14 @@ describe LuckyRecord::Model do
 
     User::BaseQuery.all.size.should eq 0
   end
+
+  describe ".fields" do
```

Comment:
I think we should name this `column_names` so it's more clear that they aren't `LuckyRecord::Field`s

Reply:
Ah mhm wasn't quite aware of `Field` being already used somewhere, thanks. ✅